src/cli/domain/get-mocked-plugins.js

```diff
@@ -25,11 +25,12 @@ var registerStaticMocks = function(mocks, logger){
   });
 };
 
-var registerDynamicMocks = function(mocks, logger){
+var registerDynamicMocks = function(mocks, ocJsonPath, logger){
```

Comment:
I think you should rename this parameter into `ocJsonLocation` as per the way you invoke the method below in line 97